crates/kubelet/src/queue.rs

```diff
@@ -0,0 +1,114 @@
+use std::collections::HashMap;
+use std::sync::Arc;
+
+use k8s_openapi::api::core::v1::Pod as KubePod;
+use kube::api::{Meta, WatchEvent};
+use log::{debug, error};
+use tokio::sync::{mpsc::Sender, watch, Mutex};
+use tokio::task::JoinHandle;
+
+use crate::handle::pod_key;
+use crate::Provider;
+
+/// A per-pod queue that takes incoming Kubernetes events and broadcasts them to the correct queue
```

Comment:
Can you break this documentation up so that the first sentence is on its own line. That makes generated documentation easier to read